libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -219,8 +246,11 @@ namespace SohImGui {
         ImGuiProcessEvent(event);
     }
 
+#ifdef ENABLE_DX11
 #define BindButton(btn, status) ImGui::Image(impl.backend == Backend::DX11 ? GetTextureByID(DefaultAssets[btn]->textureId) : (ImTextureID)(DefaultAssets[btn]->textureId), ImVec2(16.0f * scale, 16.0f * scale), ImVec2(0, 0), ImVec2(1.0f, 1.0f), ImVec4(255, 255, 255, (status) ? 255 : 0));
-
+#else
+#define BindButton(btn, status) ImGui::Image((ImTextureID)(DefaultAssets[btn]->textureId), ImVec2(16.0f * scale, 16.0f * scale), ImVec2(0, 0), ImVec2(1.0f, 1.0f), ImVec4(255, 255, 255, (status) ? 255 : 0));
```

Comment:
Just noting this section to be double checked. It looks like it is handling textures and assets, which is related to the segfault. I don't have the expertise to vet it personally, but I'll flag it just so you can confirm it should be all good.